resources/js/Components/Table/CharacterTable/CharacterTable.tsx

```diff
@@ -209,7 +210,10 @@ const CharacterTable = ({
                                             id={labelId}
                                             scope='row'
                                             padding='none'>
-                                            {row.name}
+                                            <Link
```

Comment:
so clicking a character's name to go to detail?

Reply:
yes, still need to change the color or do something so it is more obvious that we need to click the name